### ISSUE TITLE
After updating, the entity climate.anna_thermostat is gone

### ISSUE THREAD

**Issue by bouwew** [laetificat/anna-ha#6]
Today I've updated HA to 0.94.4.
After the update the entity ```climate.anna_thermostat``` did not show up in the list.
No error in the HA log.
When going in via ssh, this came up:
```
core-ssh:~# hassio ha check
Error: Testing configuration at /config
INFO:homeassistant.util.package:Attempting install of aiofiles
Failed config
  General Errors:
    - Platform not found: climate.anna
Successful config (partial)
```

I can get it to work by changing the text in configuration.yaml to:
```
anna:
  password: something
  host: something
```
After the restart there is an error in the HA log: ```no setup defined```.
Next, I change the text in configuration.yaml back to:
```
climate:
  - platform: anna
    password: something
    host: something    
```
And it is working again. Until the next HA update, I've seen this happening after every update since 0.94.0. Apparently, my proposed fixes that you accepted, did not fix the issue.

**Comment by bouwew** [comment]
Hmm, I think I found the reason for the problem to happen.

Turns out I made a mistake: I thought that the file manifest.json was required, since 0.92, but after rereading the documentation with more attention: it reads, only for integrations, not for custom_components. I'm sorry about that!

This means that the manifest.json file can be removed and the removal of the REQUIREMENTS = ['haana'] and related changes must be undone.

I'm testing this now in my local code, I want to test this with the coming 0.95.1 update. When this goes well, I will create a new pull-request.
